spec/std/callstack_spec.cr

```diff
@@ -18,7 +18,15 @@ describe "Backtrace" do
 
     # resolved file line:column
     output.should match(/callee1 at #{sample} 3:10/)
-    output.should match(/callee3 at #{sample} 15:3/)
+
+    # The first line is the old (incorrect) behaviour,
+    # the second line is the new (correct) behaviour.
+    # TODO: keep only the second one after Crystal 0.23.1
```

Comment:
Perhaps we could use the crystal compiler programatically to ensure we don't have to handle multiple compiler versions, just `master`.

Reply:
It's a good idea. I'll change it later.

Reply:
Actually, I'd prefer now to do that just now. That means the spec will depend on the compiler, and that will slow down `std_spec` a lot.